content/docs/rosa/vpn/_index.md

```diff
@@ -0,0 +1,180 @@
+---
+date: '2023-07-28'
+title: Setup a VPN Connection into a ROSA Cluster with OpenVPN
+tags: ["ROSA", "AWS"]
+authors:
+  - Kevin Collins & Kumudu Herath
+---
+
+When you configure a Red Hat OpenShift on AWS (ROSA) cluster with a private only configuration, you will need connectivity to this private network in order to access your cluster. This guide will show you how to configute an AWS Client VPN connection so you won't need to setup and configure Jump Boxes.
```

Comment:
should we say private-link instead of "private only"   ? or does this apply for regular private clusters too ?

Reply:
I updated to private-link ... it should apply to private but since we don't really recommend that anymore I'll stick to private-link